omnibus-sepolia-carina.toml

```diff
@@ -0,0 +1,547 @@
+
+# --- Configure bfp --- #
+#
+#
+
+# --- Set pyth --- #
+#
+#
+#
+[invoke.set_pyth]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setPyth"
+args = ["<%= settings.pyth_address %>"]
+
+# --- Add Rewards distributors --- #
+#
+#
+
+[invoke.create_wsteth_rewards_distributor]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "createRewardDistributor"
+extra.wsteth_rewards_distributor.event = "RewardDistributorCreated"
+extra.wsteth_rewards_distributor.arg = 0
+
+[[invoke.create_wsteth_rewards_distributor.args]]
+poolId = "<%= settings.sc_pool_id %>"
+name = "wsteth RewardDistributor"
+token = "<%= settings.wsteth_address %>"
+collateralTypes = [
+    "<%= settings.wsteth_address %>",
+    "<%= settings.weth_address %>",
+    "<%= settings.snx_address %>",
+]
+
+[invoke.create_weth_rewards_distributor]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "createRewardDistributor"
+extra.weth_rewards_distributor.event = "RewardDistributorCreated"
+extra.weth_rewards_distributor.arg = 0
+
+[[invoke.create_weth_rewards_distributor.args]]
+poolId = "<%= settings.sc_pool_id %>"
+name = "weth RewardDistributor"
+token = "<%= settings.weth_address %>"
+collateralTypes = [
+    "<%= settings.wsteth_address %>",
+    "<%= settings.weth_address %>",
+    "<%= settings.snx_address %>",
+]
+
+# --- Global Market configuration --- #
+#
+#
+
+[invoke.set_global_market_configuration]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setMarketConfiguration"
+
+[[invoke.set_global_market_configuration.args]]
+pythPublishTimeMin = 12
+pythPublishTimeMax = 60
+minOrderAge = 12
+maxOrderAge = 60
+minKeeperFeeUsd = "<%= settings.min_keeper_fee_usd %>"
+maxKeeperFeeUsd = "<%= settings.max_keeper_fee_usd %>"
+keeperProfitMarginUsd = "<%= settings.keeper_profit_margin_usd %>"
+keeperProfitMarginPercent = "<%= settings.keeper_profit_margin_percent %>"
+keeperSettlementGasUnits = 1200000
+keeperCancellationGasUnits = 600000
+keeperFlagGasUnits = 1200000
+keeperLiquidateMarginGasUnits = 1200000
+keeperLiquidationGasUnits = 1200000
+keeperLiquidationEndorsed = "<%= settings.deployer %>"                          # TODO, this is currently owner, change to a keeper
+collateralDiscountScalar = "<%= settings.collateral_discount_scalar %>"
+minCollateralDiscount = "<%= settings.min_collateral_discount %>"
+maxCollateralDiscount = "<%= settings.max_collateral_discount %>"
+utilizationBreakpointPercent = "<%= settings.utilization_breakpoint_percent %>"
+lowUtilizationSlopePercent = "<%= settings.low_utilization_slope_percent  %>"
+highUtilizationSlopePercent = "<%= settings.high_utilization_slope_percent %>"
+
+# --- Set eth oracle node id --- #
+#
+#
+
+[invoke.set_eth_oracle_node_id]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setEthOracleNodeId"
+args = ["<%= settings.eth_oracle_id %>"]
```

Comment:
I cannot find eth_oracle_id setting anywhere.

Reply:
Surprised this wasn't caught in the dry-run but good catch

Reply:
We include `"tomls/oracles/eth.toml",` which defines it as an extra:
https://github.com/Synthetixio/synthetix-deployments/blob/bfp-market-carina/tomls/oracles/eth.toml#L14

Reply:
I believe this is a bad idea to refer them as `settings` as they clearly are not settings, but values derived from events. I strongly suggest to refer them as `extras. eth_oracle_id`.

Reply:
> I believe this is a bad idea to refer them as `settings` as they clearly are not settings, but values derived from events. I strongly suggest to refer them as `extras. eth_oracle_id`.

my understanding is that extras is deprecated?

Reply:
> my understanding is that extras are deprecated?

I strongly disagree and will contest this decision